### PR TITLE
Removed poll app and added basic homepage for website.

### PR DESCRIPTION
**Pull request by sunjimmy1**
Removed the Polls App and added a basic homepage.

### CODE REVIEW

**Review by nbovee — APPROVED** [review]
Looks good to reset us to a base starting point

File: website/website/settings.py
Comment (nbovee) [inline website/website/settings.py:1]: 0.0.0.0 refers to outside hosts, best not to have that in here.

**Review by nbovee — CHANGES_REQUESTED** [review]
0.0.0.0 should be removed from allowable hosts

Please add a tests.py file back to repo to allow it to have a baseline test, ex containing:
```
def test_exists():
    assert True
```

**Comment by sunjimmy1** [comment]
Closes #2

**Review by nbovee — APPROVED** [review]
👍

**Pull request merged**